__tests__/app.test.js

```diff
@@ -231,4 +231,20 @@ describe('GET /api/articles/:article:id/comments', () => {
         })        
     });
         
+    });
+
+    describe('PATCH /api/articles/:article_id', () => {
+        test('should ',async () => {
+            const article_id=2
+            const newvote=1
+            let response=await request(app)
+            .patch(`/api/articles/${article_id}`)
+            .send({vote_inc: newvote})
+            console.log(response.body.update[0].votes)
+            expect(response.body.update[0].votes).toEqual(1)
+
+            
+            
+        });
+        
```

Comment:
I'd like to see more tests here:
- invalid article id
- valid but non-existent article id
- vote_inc not provided
- vote_inc provided but it has an invalid value